js/text-edit.picker.js

```diff
@@ -97,7 +129,7 @@ var markdownEditor = {
 
         xhr.open("PUT", url, true);
         xhr.setRequestHeader("Content-type", "application/octet-stream");
-        xhr.setRequestHeader("Authorization", "Bearer " + this.accessToken)
+        xhr.setRequestHeader("Authorization", "Bearer " + parent.accessToken.accessToken)
```

Comment:
Not a typo, parent.accessToken is an object, that includes accessToken and validForSeconds as properties.